src/funcs.c

```diff
@@ -596,10 +596,12 @@ void list_show_favorites(char* str)
 			const int length = strlen(helper);
 			for (int i = 0; i <= length - 1; i++) helper[i] = tolower(helper[i]);
 
-			if (item_games->favorite == 1 && item_games->hidden != 1 && strstr(helper, (char *)str))
+			if (item_games->favorite == 1 && item_games->hidden != 1)
 			{
-				DoMethod(app->LV_GamesList, MUIM_List_InsertSingle, item_games->title, MUIV_List_Insert_Sorted);
-				total_games++;
+				if (strstr(helper, (char *)str) || (str == NULL)) {
```

Comment:
With this fix both showing favorites on startup and filtering from the input field work just fine.

Reply:
Given this is using `strstr()` does this mean that the filtering is case-sensitive? Or do we lower-case the strings elsewhere first?

Reply:
@coldacid Good question. I will check that today how it works. If it is case sensitive, we might need a new task for next version to change it.
Thanks

Reply:
@coldacid Checked that filter and is case insensitive and can return entries even what you search is in the middle of the name.